gvi/transactions/models.py

```diff
@@ -1,3 +1,36 @@
 from django.db import models
 
-# Create your models here.
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+    number = models.CharField(max_length=50, unique=True)
+    
+    def __str__(self):
+        return self.number
+
+
+class Subcategory(models.Model):
+    name = models.CharField(max_length=50)
+    category = models.ForeignKey(Category)
+
+    def __str__(self):
+        return self.name
+
+
+class Transaction(models.Model):
+    IN = 'i'
+    OUT = 'o'
+    TYPE_CHOICES = (
+            (IN, 'Money In'),
+            (OUT, 'Money Out'),
+    )
+    transaction_type = models.CharField(max_length=5, choices=TYPE_CHOICES, default=OUT)
+    category = models.ForeignKey(Category)
+    date = models.DateTimeField()
+    subcategory = models.ForeignKey(Subcategory, blank=True)
+    comment = models.CharField(max_length=200, blank=True)
+    amount = models.CharField(max_length=50)
```

Comment:
I think it would be better to use DecimalField instead of CharField. Same thing with amount.